stock_prices/stock_prices.py

```diff
@@ -3,7 +3,18 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
+  profit = None
+  for i in range(len(prices)-1):
```

Comment:
Nice, I see that you are looping through each price and comparing with each subsequent item in the list until you find the max profit, nice solution.